kbase-extension/static/kbase/js/common/errorDisplay.js

```diff
@@ -169,26 +249,6 @@ define(['bluebird', 'uuid', 'common/html'], (Promise, Uuid, html) => {
         return errorObj;
     }
 
-    function convertInternalError(rawError) {
```

Comment:
I removed the formatting code and moved this up to be below `convertUnknownError`